packages/website/src/features/Deploy/PublishUtility.tsx

```diff
@@ -78,27 +79,35 @@ export default function PublishUtility(props: {
       isAddressEqual(publisher, wc.data?.account.address)
   );
 
-  const etherscanUrl =
-    findChain(props.targetChainId).blockExplorers?.default?.url ??
-    'https://etherscan.io';
-
   const { transports } = useProviders();
 
-  const prepareAndPublishPackage = async (registryChainId: number) => {
+  const prepareAndPublishPackage = async (
+    registryChainId: number,
+    roChainId: number
```

Comment:
What does `roChainId` mean here?

Reply:
"read only chain id"

added it because even though publishing on one chain, it is stil necessary to read from the other chain to determine if package is published or not

not a perfect fix but if people are predominantly publishing to one chain its a good solution